### PR TITLE
Update readme with authorization group instructions

### PR DESCRIPTION
**Pull request by jish**

### CODE REVIEW

**Comment by jimmycuadra** [comment]
Rather than duplicate the explanation of how to manage admins, perhaps it'd suffice to just link to the relevant section on docs.lita.io.

File: README.md [hunk 1]
```diff
@@ -4,7 +4,7 @@
 [![Code Climate](https://codeclimate.com/github/jimmycuadra/lita-retro.png)](https://codeclimate.com/github/jimmycuadra/lita-retro)
 [![Coverage Status](https://coveralls.io/repos/jimmycuadra/lita-retro/badge.png)](https://coveralls.io/r/jimmycuadra/lita-retro)
 
-**lita-retro** helps manage topics for regular team retrospective meetings.
+**lita-retro** is a [Lita](https://www.lita.io/) Handler that helps manage topics for regular team retrospective meetings.
```
Comment (jimmycuadra) [inline README.md]: Handler doesn't need to be capitalized since it's not a proper noun.

**Comment by jish** [comment]
Updated with your suggestions.

**Comment by jimmycuadra** [comment]
Thanks!

**Pull request merged**